config/application.yml.example

```diff
@@ -1,104 +0,0 @@
-# Comment out parameter below to enable syncing with lims through the REST API.
```

Comment:
Chief I think you accidentally deleted application.yml.example

Reply:
i have replaced the file and added the suggestion

Reply:
i have added the application.yml.example file